File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = dict(Counter(nums))
        sorted_items = sorted(freq.items(), key=lambda item: item[1], reverse=True)

        ret = []

        for i in range(k):
            ret.append(sorted_items[i][0])
        
        return ret

        

        

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int, int> m;

        for(auto n : nums){
            if(!m.contains(n)){
                m[n] += 1;
            }
            else{
                return true;
            }
        }
        return false;
        
    }
};
